slemicro/combustion/script

```diff
@@ -448,7 +499,7 @@ if [ "${UPDATEANDREBOOT}" = true ]; then
 	Description=Reboot if required once
 	Wants=network-online.target
 	# If services doesn't exist is ok
-	After=network.target network-online.target k3s_installer.service rancher_installer.service skip-rancher-bootstrap.service elemental_installer.service
+	After=network.target network-online.target rancher_installer.service skip-rancher-bootstrap.service elemental_installer.service
```

Comment:
Same as before, make it dependant of k3s or rke2 depending on which one has been installed.